File: Android/app/src/main/java/kalen/app/ustb_choose_course_system/model/ExamInfo.java
package kalen.app.ustb_choose_course_system.model;

/**
 * Created by kalen on 15-12-22.
 */
public class ExamInfo {
    private String className;
    private String examTime;
    private String examLocation;

    public ExamInfo(String className, String examTime, String examLocation) {
        this.className = className;
        this.examTime = examTime;
        this.examLocation = examLocation;
    }

    public String getClassName() {
        return className;
    }

    public void setClassName(String className) {
        this.className = className;
    }

    public String getExamTime() {
        return examTime;
    }

    public void setExamTime(String examTime) {
        this.examTime = examTime;
    }

    public String getExamLocation() {
        return examLocation;
    }

    public void setExamLocation(String examLocation) {
        this.examLocation = examLocation;
    }
}

File: iOS/ExamInfo.swift
//
//  ExamInfo.swift
//  ustb_choose_course_system
//
//  Created by 李鑫 on 16/1/14.
//  Copyright © 2016年 kalen blue. All rights reserved.
//

import UIKit

extension kalen.app{
    class ExamInfo: NSObject {
        var className = ""
        var examTime = ""
        var examLocation = ""
        
        init(className: String, examTime: String, examLocation: String){
            self.className = className
            self.examTime = examTime
            self.examLocation = examLocation
        }
        
    }
}

